ZenPacks/zenoss/Import4/model.py

```diff
@@ -78,45 +92,24 @@ def reportProgress(self, raw_line):
     def wipe(self):
         self.reportProgress('Wipe is done while importing zodb')
 
-    def doImport(self):
+    # TOTO: Make this a decorator
+    def _ready_to_import(self):
         if not os.path.exists(self.model_checked):
             self.reportProgress("Model backup file not validated yet. Run -c option first.")
             raise ModelImportError(Results.INVALID, -1)
 
-        if os.path.isfile(self.model_migrated):
-            os.remove(self.model_migrated)
-
         self._check_files()
 
-        # stop services accessing zodb
-        # use the provided control center IP for service controls
-        _util_cmd = "%s/imp4util.py" % self.binpath
-        if self.args.control_center_ip:
-            _util_cmd = "CONTROLPLANE_HOST_IPS=%s " % self.args.control_center_ip + _util_cmd
-
-        self.reportProgress('Stopping services ...')
-        _cmd = "%s --log-level=%s stop_svcs" % (_util_cmd, self.args.log_level)
-        self.exec_cmd(_cmd)
+    #==========================================================================
+    # Import methods
+    #==========================================================================
 
+    def database(self):
+        self._ready_to_import()
         # restore zodb
         self.reportProgress('Restoring zodb ...')
         self.restoreMySqlDb(self.zodb_sql, 'zodb', Config.zodbSocket)
 
-        # fix schema if needed
-        # self.reportProgress('No ZODB schema changes ...')
-
-        # zip up the zenpacks in the backup ZenPack dir
-        # copy it to /opt/zenoss/.ZenPack
-        self.reportProgress('Create and copying the 4.x zenpacks eggs ...')
```

Comment:
We still need to copy create and copy over the 4.x zenpacks eggs from the backup file.